README.md

```diff
@@ -5,24 +5,44 @@ This is an android application that is being created to gain experience developi
 applications written in Java, using git, solidifying programming fundamentals, and designing
 an application from markup to deployment.
 
+#### Code Rules
+  * SV Go shall be written in Java. 
+  * Application shall be written with strong variable typing.  
+     * This is to build good coding practices, and to have code written as implicit as possible. 
+  * Each line shall have a 100 character limit.  
+     * The is to improve code readability as it is
+   easier to read code in a digestible format.
```

Comment:
Bonus benefits: requires us to use meaningful variable names (ie no vars named MyImportantVarThatDoesSomethingMeaningful) and ensure code can be diffed without excessive horizontal scrolling